test/parallel/test-vm-cached-data.js

```diff
@@ -84,9 +84,12 @@ function testRejectSlice() {
 }
 testRejectSlice();
 
+const expectedError =
+  /^TypeError: options.cachedData must be a Buffer instance$/;
```

Comment:
If this is only used in one place, it probably doesn't need to go in a variable.